apps/src/blockly/addons/cdoUtils.js

```diff
@@ -261,6 +261,15 @@ export function getUserTheme(themeOption) {
   return Blockly.themes[localStorage.blocklyTheme] || themeOption || cdoTheme;
 }
 
+/**
+ * Returns a cursor type, based on the presence of an option in the browser's localStorage.
+ * @param {string} type
+ * @returns {string} one of 'default', 'basic', or 'line'
+ */
+export function getUserCursorType() {
+  return localStorage.blocklyCursor || 'default';
```

Comment:
Am I interpreting correctly that making the cursor type persistent for a given user is a follow-up task?

Reply:
Yes. Since this is just needed for testing, `localStorage` should be sufficient for getting it to work across levels/projects for a single user session. This is consistent with how we remember themes. I would imagine that we'd implement something better for user settings when we create the accessibility panel to manage these options.